lib/agenda/stop.js

```diff
@@ -18,8 +18,21 @@ module.exports = function(cb) {
   const _unlockJobs = function(done) {
     debug('Agenda._unlockJobs()');
     const jobIds = self._lockedJobs.map(job => job.attrs._id);
-    debug('about to unlock jobs with ids: %O', jobIds);
-    self._collection.updateMany({_id: {$in: jobIds}}, {$set: {lockedAt: null}}, done);
+
+    if (jobIds.length >= 1) {
```

Comment:
I meant that instead of but I'm just bikeshedding really. :-)

```
if jobs
 // unlock jobs
else
  // no jobs
```

could be
```
if !jobs
 // return no jobs

// unlock jobs
```